app/views/release_notes/_v6.0.0.html.erb

```diff
@@ -0,0 +1,6 @@
+ <h3>AACT 6.0.0<i>(November 15, 2021)</i></h3>
+
+ <p>Add Airbrake notify in StudyJsonRecord model</p>
+ <p>Check statistics each time a load is done</p>
+ <p>Run StudyJsonRecord code when doing a load</p>
+ <p>The study_statistics endpoint is used to validate data in our database</p>
```

Comment:
Needs rewording too

- "Added Airbrake notifications to StudyJsonRecord model"
- "Setup code to check study statistics each time a load is done"
- "Setup code to only use the Beta API when doing a load"
- "Setup code to compare our data for the Beta API study statistics endpoint for data validation purposes"